src/mawpy/utilities/preprocessing.py

```diff
@@ -27,7 +30,41 @@ def get_list_of_chunks_by_column(df: pd.DataFrame, column_for_chunking: str, in_
 
 def get_preprocessed_dataframe(input_file_path: str) -> pd.DataFrame:
     input_df = pd.read_csv(input_file_path)
-    input_df[UNIX_START_DATE] = input_df[UNIX_START_T].apply(
-        lambda x: datetime.utcfromtimestamp(x).strftime('%Y-%m-%d'))
-    input_df.sort_values(by=[USER_ID, UNIX_START_DATE], ascending=True, inplace=True)
+    input_df[UNIX_START_DATE] = (
```

Comment:
To avoid https://github.com/uw-ssec/MAWpy/discussions/38#discussioncomment-10036024, could you add a line above to lowercase the column names from here forward. This way we can ensure that column names are lower case and also strip any spaces, so that if user have like `orIG_lat ` it will still work since python will lowercase and strip spaces to `orig_lat`.

Reply:
[updated](https://github.com/uw-ssec/MAWpy/pull/43/commits/be044bd601ab131cfa583823ca2cb31b26553888#diff-5ba9b11cfe9b3d6270cc7cf39b857378a53b49ce9394a3e58cdeb076a876db51R33)